pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -41,14 +45,24 @@ const (
 	oldFinalizerName = "delete-knative-serving-manifest"
 )
 
+var servingManifest mf.Manifest
+
+func init() {
+	manifest, err := mf.ManifestFrom(mf.Recursive(filepath.Join(os.Getenv("KO_DATA_PATH"), "knative-serving/")))
+	if err != nil {
+		panic(err)
+	}
+	servingManifest = manifest
+}
```

Comment:
This will have to move to `ReconcileKind` eventually, right? Might as well stick it there now. The potential `panic` bothers me.

Reply:
We definitely need to cache things and I didn't want to necessarily bite that off here. I benchmarked the reading a bit and it'll be hard to improve so I think we'll need an in-mem cache.
Before we go full dynamic, I figured I leave it to a future thingy.

Reply:
Caching aside, I just meant that since we expect the spec to include a version, we'll need to construct the core manifest in the reconcile loop rather than at startup.

Reply:
Yes, but not as part of the work I'm doing. In my work stream, things are static hence I went with this approach.
The failure modes are the same as today. We logger.Fatal out today, which is equal to a panic.

Reply:
I do agree though, that `init` is mega ugly. I'll bite of a simple caching mechanism as a separate change of this that we can land independently!

Reply:
Well then why move the manifest loading into the reconciler in this PR at all?!?!

Not trying to put you on the defensive, just seems weird to use `init` for this. I'll yield to other reviewers, though.

Reply:
Mostly consistency with the ingress bits. I agree it's gross though. I'll work on it, thanks for the feedback! :)